src/base/http/requestparser.cpp

```diff
@@ -218,15 +181,11 @@ bool RequestParser::parseRequestLine(const QString &line)
     m_request.method = match.captured(1);
 
     // Request Target
-    const QUrl url = QUrl::fromEncoded(match.captured(2).toLatin1());
-    m_request.path = url.path();
-
-    // parse queries
-    QListIterator<QStringPair> i(QUrlQuery(url).queryItems());
-    while (i.hasNext()) {
-        const QStringPair pair = i.next();
-        m_request.gets[pair.first] = pair.second;
-    }
+    const QByteArray decodedUrl {QByteArray::fromPercentEncoding(match.captured(2).toLatin1())};
+    const int urlSep = decodedUrl.indexOf('?');
```

Comment:
querySep

Reply:
It separates URL into two part, so it is URL separator, isn't it?

Reply:
I was thinking this is separating out the query part, so it should reflect on its name.

Reply:
There are no other separators at this point so I changed it to more neutral "sepPos".